.github/workflows/anaconda_pkg_build_linux_alma.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and publish AlmaLinux 8 package builder images
+on:
+  #push:
+  #  branches:
+  #    - main
+  #  tags:
+  #    - 'pkg-build-*'
+  #  paths:
+  #    - 'anaconda-pkg-build/linux/alma/Dockerfile'
+  #    - '.github/workflows/anaconda_pkg_build_linux_alma.yml'
+  #pull_request:
+  #  paths:
+  #    - 'anaconda-pkg-build/linux/alma/Dockerfile'
+  #    - '.github/workflows/anaconda_pkg_build_linux_alma.yml'
```

Comment:
These were commented out on the thing you copied from just because it was completely broken (centos 7 based) -- might as well uncomment for here.

Though maybe `tags` should be `pkg-build-alma-*` 🤷

Reply:
Yep, please activate those lines so that things get automatically build in PR, on merge and on being tagged

Reply:
well, i didn't want to build them if it was going to be uploaded then potentially get pulled into the main PR

Reply:
updated